meerkat_frontend/src/js/explore/explore.js

```diff
@@ -242,32 +242,37 @@ function timelineLink(id, name, axis){
     :param string cat:
         The given category ID.
     :param object options:
-        The options object detailing how the table should be put together. 
+        The options object detailing how the table should be put together.
         Can have the following properties:
-        
+
         * **start_date** (string) - The start date by which to filter data (in ISO format).
         * **end_date** (string) - The end date by which to filter data (in ISO format).
         * **location** (number) - The location ID by which to filter the data. 
         * **strip** (boolean) - Strip empty rows from the table.
         * **colour** (boolean) - Colour the cells with a shade of the highlight colour.
           according to the proportion of the range that the value represents.
+        * **location_case** (boolean) - category breakdown for a given location (as opposed to for a given variable from another category)
 */
 function createTimeline(id, cat, options, title){
 
-	console.log( "Drawing timeline" );
-
 	//These variable will hold all the JSON data from the api, when the AJAX requests are complete.
 	var queryData, category, variable;
 
-	//Assemble the main query url according to the given options.
-	var main_query_url = api_root + '/query_variable/' + id + '/' + cat;
-	if( options.start_date && options.end_date ){
-		main_query_url += '/' + options.start_date + '/' + options.end_date;
-	}
-	main_query_url += '?use_ids=1';
-	if( options.location ){
-		main_query_url += '&only_loc=' + options.location;
-	}
+    //Assemble the main query url according to the given options.
+    var main_query_url = api_root;
+    if(options.location_case){
+        main_query_url += "/query_variable/tot_1/" + cat;
```

Comment:
I do understand what you are doing here, it makes sense and seems to work locally.  I think it is pretty important to give a title to the timeline table so it is clear what the numbers represent.  In other timelines this is done in line 305/311. For some reason it doesn't seem work with your update. Ideally the title would be something like "#Cases from <location name>" Could we try to fix this?